cypress/integration/query/search_query_spec.js

```diff
@@ -0,0 +1,24 @@
+describe("Search Query", () => {
+  beforeEach(() => {
+    cy.login();
+
+    cy.request("POST", "api/queries", {
+      name: "Users Count",
+      data_source_id: 1,
+      query: "SELECT 1"
+    });
+
+    cy.request("POST", "api/queries", {
+      name: "Dashboards Count",
```

Comment:
I wanted to add a `not contains` assertion in my test, but didn't see how to do this. 🤔

Reply:
I think the way to do it is to wrap the result table and:
```javascript
cy.getByTestIdOrAth('QuerySearchResult').should(($resultTable) => {
  expect($resultTable).to.contain('Expected Result');
  expect($resultTable).not.to.contain('Anything else');
});
```

Reply:
Exactly.
@arikfr your test will return ANY element in the page that contains that text. So if this query showed up in the "favorites" dropdown, this would always be true...

Reply:
👍 Thanks. 

I'll update the tests to do this form now.